app/assets/javascripts/beak/widgets/ractives/code-editor.coffee

```diff
@@ -6,6 +6,7 @@ window.RactiveModelCodeComponent = Ractive.extend({
   , lastCompiledCode:  undefined # String
   , lastCompileFailed:     false # Boolean
   , procedureNames:           {}
+  , autoCompleteStatus:    false
```

Comment:
Could you put type signature comments on these new variables (particularly `procedureNames`)?